packages/editor-ui/src/views/NodeView.v2.vue

```diff
@@ -1385,6 +1386,22 @@ function registerCustomActions() {
 	});
 }
 
+/**
+ * Unload
+ */
+
+function onBeforeUnload(e: BeforeUnloadEvent) {
```

Comment:
Could we extract this stuff into a `useBeforeUnload` composable, instead of duplicating this code?